tests/spec/core/utils-spec.js

```diff
@@ -8,6 +8,56 @@ describe("Core - Utils", () => {
     });
   });
 
+  async function clearCaches(){
+    const keys = await caches.keys();
+    for (const key of keys) {
+      await caches.delete(key);
+    }
+  }
+
+  describe("fetchAndCache", () => {
+    beforeEach(clearCaches);
+    it("caches a requests of different type with a 1 day default", async () => {
+      const url = location.origin + "/tests/data/pass.txt";
```

Comment:
We should add this file to `karma.conf.js`

Maybe merge in this: https://github.com/w3c/respec/blob/3c8defdee494ecc6d45e602d703c1e6ec51fa0f0/karma.conf.js#L50,L54

as 
``` js
     {
        pattern: "tests/data/**/*",
        included: false,
        served: true,
      },
```

Reply:
Good point.